compiler/src/dotty/tools/dotc/reporting/messages.scala

```diff
@@ -286,6 +286,8 @@ import transform.SymUtils._
   extends NotFoundMsg(NotAMemberID) {
     //println(i"site = $site, decls = ${site.decls}, source = ${site.typeSymbol.sourceFile}") //DEBUG
 
+    val widenedName: String  = ex"${site.widen}"
```

Comment:
We have to be very careful about this sort of things: normally, everything in an error message is lazy, this is important since we generate a lot of error messages that we never display (when finding candidates for implicit search for example), anything done eagerly in an error message can impact performance and needs to be measured.
Also, we already have some logic in type inference to stop doing inference when errors exist to make sure we don't report non-sensical messages:
https://github.com/lampepfl/dotty/blob/22c23a5cc648650bb6380dec0c11a5fdc34e1d73/compiler/src/dotty/tools/dotc/typer/Inferencing.scala#L491-L511
We should investigate why this isn't enough to handle this case.

Reply:
(one possible reason is that the logic I linked to is in `interpolateTypeVars`, but a different way to instantiate type variables is to use `isFullyDefined` / `fullyDefinedType` which doesn't go through the same codepath)

Reply:
Thanks for the tip, I took the suggested approach and it seems to work 👍